README.md

```diff
@@ -2,19 +2,39 @@
 
 1.
   * Create a Human class that takes in a name and age.
-  * Add the function `ageOneYear` that ages the human. 
+  * Add the function `ageOneYear` that ages the human.
   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
   * Create an instance of the Human class.
     * console log your humans age
     * call ageOneYear on your human
     * console log their age again.
     * call eating on your human.
-    
-2. 
+
+    class Human {
+      constructor (name, age) {
+        this.name = name;
+        this.age = age;
+      }
+      ageOneYear() {
+        this.age++;
+        return this.age;
+      }
+      eating(str) {
+        return `Mmmm, mmmm, mmmm, I'm loving it, I'm eating ${str}.`;
+      }
+    }
+
```

Comment:
The eating function should console.log that string instead of returning it. When a question asks for you to log something, it means to use console.log unless specifically told otherwise.